app/models/dossier.rb

```diff
@@ -1175,13 +1175,14 @@ def log_destroy
     app_traces = caller.reject { _1.match?(%r{/ruby/.+/gems/}) }.map { _1.sub(Rails.root.to_s, "") }
 
     payload = {
-      message: "Dossier destroyed",
+      message: "Dossier #{id} destroyed",
```

Comment:
je suis pas pour ce changement, car on perdrait la capacité à trouver rapidement dans kibana:
- un dossier supprimé (couple `message:"Dossier destroyed" and dossier_id: 123`)
- tous les dossiers supprimés par un requestId (`message "Dossier destroyed" AND request_id:ab-gh`)
(on pourrait mais faudrait faire des pirouettes avec des * et c'est bcp plus trop long quand on a tape sur 3 mois d'historique)

et je me sers souvent de ça 😬